mne/cov.py

```diff
@@ -386,66 +389,90 @@ def compute_raw_covariance(raw, tmin=None, tmax=None, tstep=0.2,
     picks : array-like of int | None (default None)
         Indices of channels to include (if None, all channels
         except bad channels are used).
+    keep_sample_mean : bool (default False)
```

Comment:
erm comes from the MGH repo of empty room recordings that are archived on a regular basis.

I think our compute_covariance_raw should match the default behavior of mne_process_raw